templates/vue/src/App.vue

```diff
@@ -47,7 +47,7 @@ export default {
     }
   },
   computed: {
-    ...mapState(["nodes"]),
+    ...mapState(["nodes", "rootId", "settings"]),
```

Comment:
Maybe we don't need this anymore? I think we should revert this whole file?

Reply:
Correct